js/stores/helpers/boundary-utils.ts

```diff
@@ -114,12 +114,22 @@ export function getBoundaryInfo(field: FieldStore, otherField: FieldStore): IBou
     }
   } else {
     // longitudinal boundary
-    const platePos = toSpherical(field.plate.center);
-    const otherPlatePos = toSpherical(otherField.plate.center);
-    // Order fields from west to east.
-    fields = platePos.lon < otherPlatePos.lon
-              ? [otherField, field]
-              : [field, otherField];
+    const { lat, lon } = toSpherical(field.absolutePos);
+    const fieldRotationAxis = field.absolutePos.clone().normalize();
+    const westTestVec = getNorthVector(lat, lon).setLength(0.5).applyAxisAngle(fieldRotationAxis, 0.5 * Math.PI);
+    const eastTestVec = getNorthVector(lat, lon).setLength(0.5).applyAxisAngle(fieldRotationAxis, -0.5 * Math.PI);
```

Comment:
Not sure if it matters, but note that the whole globe has radius 1.0. So, this vector is relatively VERY long. :) I guess in some rare cases, if plates are small enough, it could point at the plate that we don't expect here. I'll test if it works well with length = 0.1 and push the change if it does.

Reply:
Yeah, I thought it might be better to reduce the size but I wasn't sure how much. 0.1 sounds better.